Google.PowerShell/ReleaseFiles/GoogleCloudPlatform.Format.ps1xml

```diff
@@ -162,5 +162,52 @@
         </TableRowEntries>
       </TableControl>
     </View>
+    <View>
+      <Name>BigQuery Data Row</Name>
```

Comment:
Yeah, it's because it treats the rows as individual items since they're written to the pipeline as individual items.  Makes things odd for printing.

I'll add a TODO and give it the rest of the day to see if I can improve it.  It's going to matter for exporting data from jobs too, so I'd like to find a better solution if one exists.